eng/targets/Java.Common.targets

```diff
@@ -0,0 +1,87 @@
+<Project DefaultTargets="Build">
+
+  <PropertyGroup>
+    <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)' == ''">$(MSBuildProjectDirectory)\obj\</BaseIntermediateOutputPath>
+    <IntermediateOutputPath>$([MSBuild]::NormalizeDirectory('$(BaseIntermediateOutputPath)'))$(Configuration)\</IntermediateOutputPath>
```

Comment:
Could we get the Java builds using the regular artifacts/bin and artifacts/obj location❔ All exceptions from that convention make `rm artifacts -recurse` useless.